templates/calendar/index.html.twig

```diff
@@ -76,7 +76,7 @@
                   {% if d.presence is not empty %}
                     {% if d.presence.schedule | length > 1 %}
                       {% if nbSites > 1 %}
-                        <p style='margin-bottom:0;'> Présence à {{ d.presence.site }} : </p>
+                        <p style='margin-bottom:0;'> Présence / {{ d.presence.site }} : </p>
```

Comment:
/ is often used for seperate information or data. "Présence / <site_name>" is not clear for me. I read it as: Present (an information) and <site_name> (an other information).

May be we should display this with 2 different ways:
"Présent à <site_name>" (if specific site)
"Présent sur tous les sites" (if all site)

My2cents